packages/v4/src/content/design-guidelines/components/action-list/action-list.md

```diff
@@ -0,0 +1,30 @@
+---
+id: Action list
+section: components
+---
+An **action list** is a group of actions with set spacing.
+
+## Usage
+Action lists can be used in [toolbars](/components/toolbar), [modals](/components/modal), [forms](/components/form), [data lists](/components/data-list), [wizards](/components/wizard) and more. Using an action list allows you to know what spacing to use for a group of actions, depending on your use case. Although in most cases there should be 16px of spacing between buttons, there are cases where you may want more separation, such as with cancel buttons in wizards which should be placed 48px away from the rest of the actions. Using an action list ensures that these spacing guidelines are maintained and followed. 
+
+Action lists are also designed to stack buttons correctly within the space they are in.
+
+
+## Examples
+### Action list for actions in a modal
+In this example, an action list is used for action buttons inside a modal. The action list allows for 16px spacing to be maintained between the action buttons.
+
+<img src="./img/action-modal.png" alt="Action list used inside of a modal" width="446" />
+
+
+### Action list for actions in a wizard
+In this example, an action list is used for the action within a wizard. The action list allows for 16px spacing to be maintained between the main action buttons, and 48px of spacing to be maintenance between them and the cancel button.
```

Comment:
```suggestion
In this example, an action list is used for the actions within a wizard. The action list allows for 16px spacing to be maintained between the main action buttons, and 48px of spacing to be maintained between them and the cancel button.
```

It seems like "action" should be plural here — feel free to reject this suggestion if not! Also rewrote "maintenance" to its verb form, "maintained."